### ISSUE TITLE
Cannot keyboard navigate with NVDA running

### ISSUE THREAD

**Issue by Scratch2k** [freqdec/datePicker#59]
Control works fine in Chrome, I can tab in, hit ENTER to open the control, move around with arrows and select a date with ENTER. Attempt exactly the same behaviour with NVDA running and I can open with ENTER but the arrow keys don't work, I can't select a date. 

Obviously, this is a critical issue. 

I'm guessing NVDA is intercepting the arrow keys for navigation on the page. Looking in to how to fix this, if anyone has any suggestions feel free to weigh in.

**Comment by taylorchance** [comment]
Any updates on this? I just ran into the same problem now, this is pretty crucial.

**Comment by taylorchance** [comment]
Just in case anyone else runs into this issue, it is fixed by adding role="application" to a parent wrapper. The calendar itself already has this but for some reason when it comes in dynamically NVDA doesn't pick up on it.

**Comment by dotjay** [comment]
I also came across this issue. Thanks for the info.
